### ISSUE TITLE
Links at the top of accordions aren't keyboard-focusable

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#331]
From e-QIP-prototype-truetandem created by [toolness](https://github.com/toolness) : ryanhofdotgov/e-QIP-prototype-truetandem#1335  

It seems these buttons/links at the top of accordions aren't accessible via keyboards or--as far as I can tell--screen readers:

> ![2017-05-30_17-37-04](https://cloud.githubusercontent.com/assets/124687/26606103/a90e5a20-455e-11e7-97f5-5bf186277369.png)

In other words, pressing <kbd>Tab</kbd> cannot be used to move focus to these buttons and activate them.

This makes it difficult for keyboard users--who may be using the keyboard because they e.g. have an injury or disability that prevents them from making precise movements with a mouse--to use the site. It might also make things hard for non-sighted users; I'm not very good at using a screen-reader, but I couldn't figure out a way to get to these buttons in NVDA or VoiceOver either.

This may be a particularly big deal for the "Remove" buttons because it means that affected users might not be able to remove entries at all.

**Comment by bpdesigns** [comment]
looks good
